### DISCUSSION TITLE
Refetch Gravatar avatars

### DISCUSSION THREAD

**Discussion by angristan** [umputun/remark42#1047] [Ideas]
Is there a way to refetch Gravatar avatars?

The two uses cases I have in mind:

- After migrating comments from a WordPress site, all the commenters are faceless because the avatars haven't been fetched for the proxy.
- If a user changes their picture on Gravatar, when does remart42 get updated? Never? Or maybe when the users logs in again (if they ever do?)

**Comment by umputun** [comment]
> After migrating comments from a WordPress site, all the commenters are faceless because the avatars haven't been fetched for the proxy.

see https://github.com/umputun/remark42/issues/990, likely smth similar for WP

> If a user changes their picture on Gravatar, when does remart42 get updated? Never? Or maybe when the users logs in again (if they ever do?)

Gravatars fetched on login

**Reply by angristan** [reply]
> see #990, likely smth similar for WP

Thanks, I missed that in the search 🙂
